pages/index.js

```diff
@@ -11,13 +11,13 @@ export default function Home({ entries }) {
     <>
       <Head>
         <title>Juani Gallo - Blog</title>
-        <meta name='description' content="Juani Gallo's blog" />
+        <meta name='description' content='Juani Gallos blog' />
         <link rel='icon' href='/favicon.ico' />
       </Head>
       <Nav />
       <div className={styles.container}>
         <section className={styles.entries}>
-          {entries.length == 0 && "There are no entries"}
+          {entries.length == 0 && 'There are no entries'}
```

Comment:
Please fix the linting in here

Reply:
It's already in single quote, should change it to double quote?